src/cpp/include/openvino/genai/audio_utils.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+#include "openvino/genai/visibility.hpp"
+
+namespace ov {
+namespace genai {
+namespace utils {
+namespace audio {
+bool OPENVINO_GENAI_EXPORTS read_wav(const std::string& fname,
+                                     std::vector<float>& pcmf32,
+                                     std::vector<std::vector<float>>& pcmf32s,
+                                     bool stereo = false);
```

Comment:
do you think it can be utils of samples it self instead of part of the library? 
E.g. GenAI accepts already read / parsed data as ov::Tensor and performs inference of pipeline.

Reply:
We didn't decide what to do with this reader so far so current reader place is just a proposal.
I'm ok to move it to the samples utils. But this would require an extra steps (install the reader package, read wav file) from the library end-user perspective.

Reply:
Regarding passing `ov::Tensor` as an input. The current audio preprocessing is embedded to generate method. We could extract it to `WhisperProcessor` and do the preprocessing before `generate` call in the client code. Like HF does:

```python
# Load the Whisper model in Hugging Face format:
processor = WhisperProcessor.from_pretrained("openai/whisper-tiny.en")
model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-tiny.en")

# Use the model and processor to transcribe the audio:
input_features = processor(
    audio_sample["array"], sampling_rate=audio_sample["sampling_rate"], return_tensors="pt"
).input_features

# Generate token ids
predicted_ids = model.generate(input_features)
```

Reply:
Agreed to do feature extraction inside pipeline. To put file reading to samples utils. Done.